Winapp2.ini

```diff
@@ -5986,9 +5985,8 @@ FileKey6=%Documents%\DVDFab*\Temp\Record\webService|*.*|RECURSE
 FileKey7=%Documents%\DVDFab*\Temp\ReportCrash|*.*|RECURSE
 FileKey8=%Documents%\DVDFab*\Temp\updateTemp\adPixtureTemp|*.*|RECURSE
 FileKey9=%Documents%\DVDFab*\Temp\updateTemp\updateLog|*.*|RECURSE
-FileKey10=%Documents%\DVDFab\Log|*.*|RECURSE
-FileKey11=%Documents%\DVDFab\Temp|*.*|RECURSE
-FileKey12=%Documents%\PcSetup|*.*|RECURSE
+FileKey10=%Documents%\DVDFab\Temp|*.*|RECURSE
```

Comment:
It looks like if this were modified to read 

%Documents%\DVDFab\Temp|*.*|RECURSE

it would supersede the keys above it

FileKey3=%Documents%\DVDFab*\Temp\adPixtureTemp|*.*|RECURSE
FileKey4=%Documents%\DVDFab*\Temp\MainMovie|*.*|RECURSE
FileKey5=%Documents%\DVDFab*\Temp\MediaServer|*.*|RECURSE
FileKey6=%Documents%\DVDFab*\Temp\Record\webService|*.*|RECURSE
FileKey7=%Documents%\DVDFab*\Temp\ReportCrash|*.*|RECURSE
FileKey8=%Documents%\DVDFab*\Temp\updateTemp\adPixtureTemp|*.*|RECURSE
FileKey9=%Documents%\DVDFab*\Temp\updateTemp\updateLog|*.*|RECURSE

Would this be a valid change for this application?

Reply:
Additionally, we can probably merge [DVDFab Media Player 2 *] into this entry

Reply:
I don't use DVDFab but I think FileKey 10 is for version 1.0 of DVDFab. Maybe a user of DVDFab can tell if the FileKeys 3-9 are really needed for newer versions?